LineSDKStarterObjC/Podfile

```diff
@@ -1,10 +1,10 @@
 # Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '10.0'
 
 target 'LineSDKStarterObjC' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   # use_frameworks!
 
   # Pods for LineSDKStarterObjC
-  pod 'LineSDK', '~> 4.1.1'
+  pod 'LineSDK', '~> 5.0.0'
```

Comment:
Maybe we should prefer `5.0` instead of `5.0.0` here. `5.0` allows us to fetch and install `5.0.1`, `5.1`, `5.2` etc. But `5.0.0` would only allow `5.0.1` (users cannot upgrade to `5.1.x` or `5.2.x` later)

Reply:
Okay, I changed the Podfile to use `5.0`